automod/triggers.go

```diff
@@ -180,6 +188,12 @@ func (wl *WordListTrigger) UserSettings() []*SettingDef {
 			Key:  "ListID",
 			Kind: SettingTypeList,
 		},
+		{
+			Name:    "Also match Zalgo and confusable characters",
```

Comment:
Make it simple for a average user? What does Zalgo and confusable characters here mean?